docs/sql-ref-syntax-aux-resource-mgmt-add-archive.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: global
+title: ADD ARCHIVE
+displayTitle: ADD ARCHIVE
+license: |
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+---
+
+### Description
+
+`ADD ARCHIVE` can be used to add an archive file to the list of resources. The given archive file should be one of .zip, .tar, .tar.gz, .tgz and .jar. The added archive file can be listed using [LIST ARCHIVE](sql-ref-syntax-aux-resource-mgmt-list-archive.html).
+
+### Syntax
+
+```sql
+ADD ARCHIVE file_name
```

Comment:
`ADD { ARCHIVE | ARCHIVES }`?

Reply:
`ADD ARCHIVES` is not supported as well as `ADD FILES` and `ADD JARS` for now.

Reply:
> ADD ARCHIVES is not supported as well as ADD FILES and ADD JARS for now.

Ah, I see. Could you add tests for this negative case somewhere?

Reply:
Ah, I see. I missed that. Thanks.